openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -828,8 +833,27 @@ class c17000(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "Medical expense deduction"
+    unit = "currency-USD"
     documentation = """Sch A: Medical expenses deducted (component of pre-limitation c21060 total)"""
 
+    def formula(tax_unit, period, parameters):
+        medical = parameters(period).tax.deductions.itemized.medical
+        has_aged = (tax_unit("age_head", period) >= 65) | (
```

Comment:
```suggestion
        aged_head = tax_unit("age_head", period) >= 65
        aged_spouse = tax_unit("tax_unit_is_joint") & (tax_unit("age_spouse", period) >= 65)
        has_aged = aged_head | aged_spouse
```
btw I think the `tax_unit_is_joint` is redundant, since `age_spouse` is 0 if there's no spouse, but we can leave it for now and confirm later.

Reply:
Yeah I think you're right - will leave in for now and come back to later. I also would prefer to write this statement (and others like it) in an explicit way, maybe using some OpenFisca feature and `age`.